clang/test/dpct/driver_context.cu

```diff
@@ -35,7 +35,7 @@ int main(){
   // CHECK: ctx = dpct::select_device(device);
```

Comment:
Should use push_device here because this API will push the context into the stack.

Reply:
Done